k-distribution/tutorial/1_k/3_lambda++/lesson_1/exercises/from-call-CC-to-callcc/Makefile

```diff
@@ -1,4 +1,3 @@
 MAKEFILE_PATH := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 include $(MAKEFILE_PATH)/../../Makefile
 KOMPILE_FLAGS=--main-module CALLCC
-KOMPILE_BACKEND?=java
```

Comment:
Why was this removed? Are we using LLVM backend for this one now intentionally?

Reply:
It was already using the llvm backend, because of a bug in the way the makefile was written. Since the underlying lesson also uses the llvm backend, I simply made this explicit.